src/Analyzer.BicepProcessor/Analyzer.BicepProcessor.csproj

```diff
@@ -9,8 +9,8 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Azure.Bicep.Core" Version="0.12.40" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Azure.Bicep.Core" Version="0.14.85" />
```

Comment:
This was also in Analyzer.Core.NuGet\Analyzer.Core.nuspec, do we need to remove it from there or also update it?